docs/general/builtins/scalar.rst

```diff
@@ -2748,6 +2748,30 @@ Example:
     SELECT 1 row in set (... sec)
 
 
+.. _format_type:
+
+``format_type(integer, integer)``
+---------------------------------
+
+Returns the type name of a type. The first argument is the ``OID`` of the type.
+The second argument is the type modifier. This function exits for PostgreSQL
+compatibility and the type modifier is always ignored.
+
+Returns: ``text``
+
+Example:
+
+::
+
+    cr> SELECT pg_catalog.format_type(25, null) AS name;
```

Comment:
lgtm

one note: shall we document what is returned when a type is not found?

Reply:
yep might make sense. I added a sentence about it